core/src/main/resources/jenkins/agents/CloudSet/index.jelly

```diff
@@ -120,5 +131,12 @@ THE SOFTWARE.
         </j:otherwise>
       </j:choose>
     </l:main-panel>
+    <script>
+
+      document.querySelectorAll('tbody')
```

Comment:
Before this can be merged this will need to be moved to it's own JavaScript file:

> New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).